lib/src/open_url_base.dart

```diff
@@ -8,6 +8,11 @@ import 'dart:io';
 /// A process is spawned to run that utility, with the [ProcessResult]
 /// being returned.
 Future<ProcessResult> openUrl(String url) {
+  if (Platform.isWindows) {
+    /// This is needed because ampersand has to be escaped with carret on Windows shell,
+    /// otherwise opened URL will be trimmed by first ampersand
+    url = url.replaceAllMapped(RegExp("([^^])&"),(m) => "${m[1]}^&");
```

Comment:
Can you explain why you're using this replacement method, instead of something like:

```dart
replaceAll('&', '^&')
```

Also, is there any obvious character that may need to be skipped on Windows? I think the Dart `run` function already takes care of these things on Mac/Linux, but I will test a few things here.

Reply:
But who is going to give us an URL with already escaped ampersands?
Hm, maybe Windows users who worked around the problem already :D.
I guess that's ok, will merge... thanks.